patches/minecraft/net/minecraft/entity/item/EntityItem.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/entity/item/EntityItem.java
 +++ ../src-work/minecraft/net/minecraft/entity/item/EntityItem.java
-@@ -38,6 +38,11 @@
+@@ -5,7 +5,6 @@
+ import net.minecraft.entity.Entity;
+ import net.minecraft.entity.MoverType;
+ import net.minecraft.entity.player.EntityPlayer;
+-import net.minecraft.init.Items;
```

Comment:
This also needs keeping out of the diff.

Reply:
Are you referring to the removed net.minecraft.init.Items; import?
Everything else is the same...

Reply:
Yes, do not change imports *at all* in patches.

Reply:
This should be resolved now